packages/cashier/src/Stores/__tests__/on-ramp-store.spec.js

```diff
@@ -0,0 +1,377 @@
+import OnRampStore from '../on-ramp-store';
+import OnrampProviders from 'Config/on-ramp-providers';
+import { waitFor } from '@testing-library/react';
+
+let changelly_provider, on_ramp_store, onramp_providers, root_store, WS;
```

Comment:
@george-usynin-binary Could you consider consistency for naming `onramp`, if you are considering it as one word do it in all places if not ...

Reply:
@Bahar-FS sure, thx for the notice